README.md

```diff
@@ -1,364 +1,459 @@
+#14 sha256:70b2b4c80178ec0c5bfd4f0bc750e6294b0c3e971dfae46c66d527b476c24be8
+#14 transferring context: 40.43kB 0.1s
+#14 transferring context: 9.29MB 0.6s done
+#14 DONE 0.6s
 
-To exit, you will need to type <b>exit</b> at the bash prompt (as opposed to Ctrl + C)
+#13 [build-env 2/6] WORKDIR /app
+#13 sha256:05acf7dc78e229ecdfa0464d9bdfba70ce25016d95856acf1a6b234532c4587e
+#13 CACHED
 
-## Uploading to DockerHub
-Once you have your docker container running locally, you will want to put it in a repository so that it can be used. You have many options for this and usually it depends on a couple of things. First, is this an image that will be open source and be used as either a base image or a starter image? Or is this a personal/company image that needs to stay private.
+#15 [build-env 3/6] COPY *.csproj ./
+#15 sha256:cd17fe6b9a7581f85e0e1c0042db2e70d221dc67a6927d7820da2cd994bfbeef
+#15 CACHED
 
+#16 [build-env 4/6] RUN dotnet restore
+#16 sha256:c5cee86bf2a7f1b137c6ba94fff800745a0f85892c3628b4f652ce4ae770b86c
+#16 CACHED
 
+#17 [build-env 5/6] COPY . ./
+#17 sha256:2b0d3e20073e7fddaf62936b3fe19fe1cc4bed8e3651a8d190943e4af86cee5b
+#17 DONE 0.2s
 
+#18 [build-env 6/6] RUN dotnet publish -c Release -o out
+#18 sha256:505fddea54d3a50218c09b2d5daed79b43c902cc45bb3b337ec89278f48eb18e
+#18 0.566 Microsoft (R) Build Engine version 17.2.0+41abc5629 for .NET
+#18 0.566 Copyright (C) Microsoft Corporation. All rights reserved.
+#18 0.566
+#18 1.841   Determining projects to restore...
+#18 3.683   Restored /app/TodoApi.csproj (in 843 ms).
+#18 8.363   TodoApi -> /app/bin/Release/net6.0/TodoApi.dll
+#18 8.478   TodoApi -> /app/out/
+#18 DONE 8.6s
 
-
-
-If your container is meant to be consumed (ala base image) DockerHub is probably the best place to host your container.  It allows easy access to it and works seamlessly in the Dockerfile.  If your container is a private container then you need to know where they are running. Your containers should be registered in the same cloud you use for hosting them. 
+#11 [stage-1 2/3] WORKDIR /app
+#11 sha256:5790d9d246621300d2780afef58e515532a82e570b8580baf68a70f6b5e8a4cf
+#11 CACHED
+
+#19 [stage-1 3/3] COPY --from=build-env /app/out .
+#19 sha256:523fdcb93ec048adf029f85134458e7ead7e3a5146224cef071911ac449eff09
+#19 DONE 0.2s
+
+#20 exporting to image
+#20 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
+#20 exporting layers
+#20 exporting layers 0.2s done
+#20 writing image sha256:7e28663f832381c26e1ee7fc058446d2895aa1fe83c91212010859cf45c1bcf9 0.0s done
+#20 naming to docker.io/library/todov1 0.0s done
+#20 DONE 0.3s
+
+Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
+```
+
+Next we can run the container:
+
+```bash
+docker run -it --rm -p 5000:5000 -e "ASPNETCORE_URLS=http://+:5000" --name To_Do_App todov1
+```
+
+> **_NOTE:_** Bash users might get an error like "the input device is not a TTY. If you are using mintty, try prefixing
```

Comment:
This note is all new. Do you want to keep it? It's specific guidance for bash users that wasn't terribly hard to figure out